test/recipes/90-test_store.t

```diff
@@ -44,21 +44,32 @@ my @generated_files =
      "rsa-key-pkcs8-pbes2-sha1.pem", "rsa-key-pkcs8-pbes2-sha1.der",
      "rsa-key-sha1-3des-sha1.p12", "rsa-key-sha1-3des-sha256.p12",
      "rsa-key-aes256-cbc-sha256.p12",
-     "rsa-key-md5-des-sha1.p12",
-     "rsa-key-aes256-cbc-md5-des-sha256.p12",
      "rsa-key-pkcs8-pbes2-sha256.pem", "rsa-key-pkcs8-pbes2-sha256.der",
-     "rsa-key-pkcs8-pbes1-md5-des.pem", "rsa-key-pkcs8-pbes1-md5-des.der",
+    );
+my $use_md5 = 0; # does not work: !disabled("md5");
```

Comment:
Why does `!disabled("md5")` not work?

Reply:
For some reason the "no-md5" config option is not (anymore?) usable.

Moreover, even if md5 is available calls like
```
../../../apps/openssl pkcs12 -inkey rsa-key-pkcs8.pem -in rsa-cert.pem
 -passout 'pass:password' -chain -CAfile cacert.pem -export -macalg SHA1
 -certpbe pbeWithMD5AndDES-CBC
 -keypbe pbeWithMD5AndDES-CBC -out rsa-key-md5-des-sha1.p12
```
may fail apparently because pbeWithMD5AndDES-CBC is deprecated.
How can one test at this point if pbeWithMD5AndDES-CBC is enabled?

Reply:
Again my question: how to check in Perl whether pbeWithMD5AndDES-CBC is enabled?

Reply:
`pbeWithMD5AndDES-CBC` per se isn't deprecated as far as I can tell from finding it in the source.  However, DES-CBC is in the legacy provider, so you need to ensure it gets loaded (you want to use the config file `test/default-and-legacy.cnf` to have that).  Additionally, `des` may be disabled, sso if you want to check anything, it should be that.

Reply:
Interesting hint. Since the pkcs12 app does not have a `-config` option I tried this way:
```
OPENSSL_CONF=test/default-and-legacy.cnf V=1 make  test TESTS=test_store | grep pbeWithMD5AndDES-CBC
```
but I still get
```
../../../util/wrap.pl ../../../apps/openssl pkcs12 -inkey rsa-key-pkcs8.pem -in rsa-cert.pem -passout 'pass:password' -chain -CAfile cacert.pem -export -macalg SHA1 -certpbe pbeWithMD5AndDES-CBC -keypbe pbeWithMD5AndDES-CBC -out rsa-key-md5-des-sha1.p12 => 1
```

Reply:
In perl, you would set it like this:

``` perl
$ENV{OPENSSL_CONF} = srctop_file("test", "default-and-legacy.cnf");
```

Reply:
> In the perl recipe, you would set it like this:
> 
> ```perl
> $ENV{OPENSSL_CONF} = srctop_file("test", "default-and-legacy.cnf");
> ```

Thanks for the hint. 
Yet it took me a while to realize that, for whatever reason, the configuration file is ignored in the tests, but calling `openssl pkcs12 -provider default -provider legacy` does the trick.

Reply:
No, `OPENSSL_CONF` isn't ignored, it's *overwritten* by the test runner:

https://github.com/openssl/openssl/blob/00da0f69890874feaa555fafb99b967b861e9118/test/run_tests.pl#L30-L34